src/Utilities/myBar.js

```diff
@@ -0,0 +1,30 @@
+import React from 'react'
+import AppBar from '@material-ui/core/AppBar';
+import Toolbar from '@material-ui/core/Toolbar';
+import AccountCircleIcon from '@material-ui/icons/AccountCircle';
+import Typography from '@material-ui/core/Typography';
+import { IconButton } from '@material-ui/core';
+
+
+export default class MyBar extends React.Component {
+    constructor(props){
+        super(props);
+        this.state = {};
```

Comment:
Hmm seems like state is never accessed by the class.
I think you may be able to remove it, unless there's some implicit functionality i'm unaware of.